app/src/main/AndroidManifest.xml

```diff
@@ -12,7 +15,13 @@
         android:supportsRtl="true"
         android:theme="@style/Theme.AndroidLearning"
         tools:targetApi="31">
-
+        <activity
+            android:name=".PressNotification"
+            android:exported="false">
+            <meta-data
+                android:name="android.app.lib_name"
+                android:value="" />
```

Comment:
why its defined?

Reply:
Maybe when I create new activity it came automatically, then I searched on internet it is key value pair we can store api key etc here

Reply:
Does it required here?

Reply:
nope